packages/login/src/components/passwordUpdate/PasswordUpdateDialog/PasswordInput.js

```diff
@@ -23,7 +25,8 @@ PasswordInput.propTypes = {
   onChange: React.PropTypes.func.isRequired,
   readOnly: React.PropTypes.bool,
   onKeyDown: React.PropTypes.func,
-  autoFocus: React.PropTypes.bool
+  autoFocus: React.PropTypes.bool,
+  formGroupStateClass: React.PropTypes.string
```

Comment:
thats exactly the point. i want to set a class on a react component -> use className as you know it from JSX
Our tocco-ui components also us className : https://tocco.github.io/tocco-client/


anyway. it seams wrong the give a css class as input. why not set flags and PasswordInput decides what to do with it. So instead of className/formGroupStateClass add prop isValid/hasError (or whatever)